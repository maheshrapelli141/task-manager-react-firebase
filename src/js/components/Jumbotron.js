import React from 'react';
import PropTypes from 'prop-types';

const Jumbotron = ({ children }) => (
  <div className="jumbotron jumbotron-fluid">
    <div className="container">{children}</div>
  </div>
);

Jumbotron.propTypes = {
  children: PropTypes.node.isRequired
};

export default Jumbotron;
