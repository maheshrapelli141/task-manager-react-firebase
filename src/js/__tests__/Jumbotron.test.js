import React from 'react';
import { shallow } from 'enzyme';

import Jumbotron from '../components/Jumbotron';

const wrapper = shallow(<Jumbotron>Sample Header</Jumbotron>);

describe('Jumbotron component', () => {
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
