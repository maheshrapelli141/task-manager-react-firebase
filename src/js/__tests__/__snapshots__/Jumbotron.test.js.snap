// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbotron component should render correctly 1`] = `
<div
  className="jumbotron jumbotron-fluid"
>
  <div
    className="container"
  >
    Sample Header
  </div>
</div>
`;
